### DISCUSSION TITLE
Welcome to ZY_Player_flutter Discussions!

### DISCUSSION THREAD

**Discussion by xiaojia21190** [xiaojia21190/ZY_Player_flutter#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Storm-J** [comment]
Hi, is this software still available? I can't download the zip package

**Reply by xiaojia21190** [reply]
https://jizutang.oss-cn-beijing.aliyuncs.com/download/upload/%E8%99%B1%E5%AD%90%E8%81%9A%E5%90%88.apk

**Reply by javon4016** [reply]
很好的一款app，但是软件有2个问题，期望作者能更新修复
1、进入专题页面，不管影视、剧集或者动漫，点击播放提示网络未连接
2、听书资源全部时效了

**Reply by xiaojia21190** [reply]
使用的影视资源网站，没有资源
听书资源，再找找

**Comment by xiaojia21190** [comment]
听书资源问题已解决

**Reply by javon4016** [reply]
感谢回复和修复软件，是更新了新版吗？之前下载的还是加载不了听书资源

**Reply by xiaojia21190** [reply]
好了

**Reply by javon4016** [reply]
已经可以加载出来，但是点击播放后全部提示“当前歌曲仅在KUWO音乐最新版播放...”

**Reply by xiaojia21190** [reply]
那就没办法了. 大部分应该还可以
